src/Sarif.Converters/ContrastSecurityConverter.cs

```diff
@@ -762,10 +788,18 @@ private string BuildSourcesString(HashSet<Tuple<string, string>> sources)
             var sb = new StringBuilder();
             foreach (Tuple<string, string> tuple in sources)
             {
-                if (tuple.Item1 == null || tuple.Item2 == null) { continue; }
```

Comment:
>tuple [](start = 20, length = 5)

This code didn't work when there was no name for the source, for example, when the tuple was `( , body)` or `( , other)`.

It also didn't format well if there was more than one source (it squished the sources together with no white space). With this change, the two sources above (which are real data from the `path-traversal` rule) format together as `body, other`.